### PR TITLE
Change profile picture and encode in 64Base -> profile-pic.component.ts

### PR DESCRIPTION
**Pull request by TeoReu**
I added feed, chat and get-matched components (ignore). CSS of the page, and profile-pic component which will be used to change profile picture.

### CODE REVIEW

File: frontend/app/src/app/profile/profile-pic/profile-pic.component.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+          }
+          const reader = new FileReader();
+          reader.onload = (e: any) => {
+              const image = new Image();
+              image.src = e.target.result;
+              image.onload = rs => {
+                  const img_height = rs.currentTarget['height'];
+                  const img_width = rs.currentTarget['width'];
+
+                  console.log(img_height, img_width);
```
Comment (chripell) [inline frontend/app/src/app/profile/profile-pic/profile-pic.component.ts:53]: This sounds as a debugging leftover.

File: frontend/app/src/app/profile/profile-pic/profile-pic.component.ts [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+
+                  if (img_height > max_height && img_width > max_width) {
```
Comment (chripell) [inline frontend/app/src/app/profile/profile-pic/profile-pic.component.ts:56]: || ?

File: frontend/app/src/app/profile/profile-pic/profile-pic.component.ts [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+                      this.imageError =
+                          'Maximum dimentions allowed ' +
+                          max_height +
+                          '*' +
```
Comment (chripell) [inline frontend/app/src/app/profile/profile-pic/profile-pic.component.ts:60]: x

**Review by chripell — APPROVED** [review]
Please fix the small problem (|| instead of &&) before merging.

**Review by Omn1 — APPROVED** [review]

File: frontend/app/src/app/profile/profile-pic/profile-pic.component.ts [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+          reader.onload = (e: any) => {
+              const image = new Image();
+              image.src = e.target.result;
+              image.onload = rs => {
+                  const img_height = rs.currentTarget['height'];
+                  const img_width = rs.currentTarget['width'];
+
+                  if (img_height > max_height || img_width > max_width) {
+                      this.imageError =
+                          'Maximum dimentions allowed ' +
```
Comment (Omn1) [inline frontend/app/src/app/profile/profile-pic/profile-pic.component.ts:57]: I may sound like a nerd, but this should be spelled "dimensions"

Comment (TeoReu) [reply frontend/app/src/app/profile/profile-pic/profile-pic.component.ts]: :^)  thanks for spotting that out.

**Pull request merged**